index.d.ts

```diff
@@ -0,0 +1,313 @@
+
+            /**
+             * Your app's Notification Content Extension with the matching category will be invoked in order to
+             * display custom notification UI.
+             */
+            category: string;
+
+            /**
+             * The localization key of a string present in your app's Localizable.strings file.
+             *
+             * Use this parameter to localize the notification body. Refer to the APNs documentation for more
+             * information.
+             */
+            loc_key: string;
+
+            /**
+             * The replacement strings to substitute in place of the %@ placeholders of the localization string
+             * matching the specified loc_key.
+             *
+             * Use this parameter to localize the notification body. Refer to the APNs documentation for more
+             * information.
+             */
+            loc_args: Array<string>;
+
+            /**
+             * The localization key of a string present in your app's Localizable.strings file.
+             *
+             * Use this parameter to localize the notification title. Refer to the APNs documentation for more
+             * information.
+             */
+            title_loc_key: string;
+
+            /**
+             * The replacement strings to substitute in place of the %@ placeholders of the localization string
+             * matching the specified title_loc_key.
+             *
+             * Use this parameter to localize the notification title. Refer to the APNs documentation for more
+             * information.
+             */
+            title_loc_args: Array<string>;
+
+            /**
+             * Indicate the importance and delivery timing of a notification on iOS 15+, with possible values
+             * of passive, active, time-sensitive, or critical.
+             *
+             * Defaults to active. Anything above active requires capabilities to be enabled in your Xcode
+             * project. Refer to the APNs documentation for more information.
+             */
+            interruption_level: string;
+        };
+    }
+
+    interface NotificationStatus {
+        /** The creation date of the push notification (unix timestamp). */
+        date: number;
+
+        /** The push notification payload data. */
+        payload: unknown;
+
+        /** The push notification expiration date (unix timestamp). */
+        expiration: number;
+
+        /**
+         * An array of device tokens that have not received the push notification yet.
+         * Limited to a maximum of 1,000 tokens.
+         */
+        pending_devices: Array<string>;
+    }
+
+    interface DeviceInfo {
+        /**  Metadata object for the device. */
+        device: {
+            /**  The device's registration date (unix timestamp). */
+            date: number;
+
+            /**  The device platform string identifier. */
+            platform: 'android' | 'ios' | 'web';
```

Comment:
@pushy Are this values the correct? It's not specified on the docs

Reply:
Nice catch. There are two more platforms: `electron` and `python`.